server/game/cards/06-WoE/SunkCost.js

```diff
@@ -0,0 +1,32 @@
+const Card = require('../../Card.js');
+
+class SunkCost extends Card {
+    // Play: Choose a house. Your opponent discards a random card from
+    // their hand. If that card belongs to the chosen house, make a
+    // token creature.
+    setupCardAbilities(ability) {
+        this.play({
+            condition: (context) => !!context.player.opponent,
+            target: {
+                mode: 'house',
+                houses: (context) => context.player.opponent.houses
+            },
+            effect: 'make a token creature if {1} randomly discards a card matching {2}',
+            effectArgs: (context) => [context.player.opponent, context.house],
+            effectAlert: true,
```

Comment:
I do not think `effectAlert` is needed here

Reply:
I disagree -- it is the same type of effect as Allusions of Grandeur, which does have the `effectAlert` (I copied this from there).  But if you have a reason to think we should treat Sunk Cost differently I'm happy to change it.

Reply:
Oops sorry, I was confusing this one with Watch Your Step.  You're right, I removed it.